tests/utils.go

```diff
@@ -3095,6 +3095,30 @@ func SkipIfNoCmd(cmdName string) {
 	}
 }
 
+func RunSystemctlCommand(args ...string) (string, string, error) {
+	cmdName := "systemctl"
+	cmd := exec.Command(cmdName, args...)
+
+	commandString := commandString(cmdName, args...)
+
+	var output, stderr bytes.Buffer
```

Comment:
lines 3104-3119 looks like a copy paste of [this](https://github.com/kubevirt/kubevirt/blob/b350d371da520fc328ffedc927c58dedd1c44f4f/tests/utils.go#L3199-#L3216).
Can you please extract to a method and reuse in both places?